Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Volume_plane.h

```diff
@@ -219,7 +232,7 @@ class Volume_plane : public Volume_plane_interface, public Tag {
 
   void drawSpheres(x_tag) const
   {
-      int max_dim = (std::max)((std::max)(adim_, bdim_ ), cdim_);
+      update_maxDim();
```

Comment:
I do not understand why `max_dim` become a mutable member data, instead writing;
```C++
double max_min = compute_max_dim();
```